visma/functions/variable.py

```diff
@@ -83,7 +83,26 @@ def __radd__(self, other):
 
     def __add__(self, other):
         from visma.functions.constant import Constant
-        if isinstance(other, Constant):
+        if isinstance(other, Variable):
+            if self.power == other.power:
```

Comment:
I think check for `self.value == other.value` must also be made as well.

Reply:
Yes it should be, rectified that.